rllib/algorithms/a3c/tests/test_a3c.py

```diff
@@ -27,7 +27,7 @@ def test_a3c_compilation(self):
         num_iterations = 2
 
         # Test against all frameworks.
-        for _ in framework_iterator(config, with_eager_tracing=True):
+        for _ in framework_iterator(config, with_eager_tracing=False):
```

Comment:
@gjoliver @kouroshHakha Not sure why this test did not fail initially, but A3C seems error with eager tracing as well, right @krfricke ?